e2e/tests/009.sh

```diff
@@ -88,14 +82,14 @@ while [[ $retries -gt 0 ]]; do
 
     modified=false
     info "Pushing update"
-    output=$(kpt alpha rpkg push -n default "$smf_pkg_rev" $ws >/dev/null 2>&1)
+    output=$(kpt alpha rpkg push -n default "$smf_pkg_rev" $ws 2>&1)
```

Comment:
@johnbelamaric @vjayaramrh changes on lines 85, 92 and 102 will fix some CI issues, I think the previous change was sending stdout and stderr to /dev/null